spec/requests/movies_spec.rb

```diff
@@ -0,0 +1,334 @@
+require 'rails_helper'
+
+RSpec.describe "/movies", type: :request do
+  let(:user) { create :user }
+  let(:manager) { create :user, email: "testmanager@test.com", role: 1 }
```

Comment:
You can also check here if this notation works
```suggestion
  let(:manager) { create :user, email: "testmanager@test.com", role: :manager}
```
If this does not work then you can create a let variable named manager_role (or sth) with value 1. It will be easier to understand what exactly is role nr 1. But you named this varaible (manager) well for me it is enough because the value of role is not so important in scope of this test. So you can do with this whatever you want 👍